src/rpm/builder.rs

```diff
@@ -277,9 +279,10 @@ impl RPMBuilder {
         let hash_result = hasher.finalize();
         let sha_checksum = hex::encode(hash_result); // encode as string
         let entry = RPMFileEntry {
+            // file_name() should never fail because we've checked the special cases already
             base_name: pb.file_name().unwrap().to_string_lossy().to_string(),
             size: content.len() as u32,
-            content: Some(content),
```

Comment:
It doesn't appear that we're ever using the `None` value, we don't create any `RPMFileEntry` with default values that I can tell, nor do any functoins construct one without passing in a `content`.

There might be some reason for this I am unaware of, though.